packages/datagateway-search/src/search/searchButton.component.tsx

```diff
@@ -91,26 +103,31 @@ class SearchButton extends React.Component<SearchButtonCombinedProps> {
   public handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
     if (this.props.dataset === true) {
       let datasetParams = this.urlParamsBuilder('Dataset');
-      this.sendRequest(datasetParams);
+      this.luceneRequest(datasetParams);
     }
     if (this.props.datafile === true) {
       let datafileParams = this.urlParamsBuilder('Datafile');
-      this.sendRequest(datafileParams);
+      this.luceneRequest(datafileParams);
     }
     if (this.props.investigation === true) {
       let investigationParams = this.urlParamsBuilder('Investigation');
-      this.sendRequest(investigationParams);
+      this.luceneRequest(investigationParams);
     }
   };
 
-  public async sendRequest(queryParams: LuceneParameters): Promise<void> {
+  public async luceneRequest(queryParams: LuceneParameters): Promise<void> {
```

Comment:
I'd prefer if this was `sendLuceneRequest`/`fetchLuceneResults` or something similarly verb-y rather than noun-y.